src/Microsoft.AspNetCore.Authentication.Cookies/Events/CookieResultContext.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Http;
+
+namespace Microsoft.AspNetCore.Authentication.Cookies
+{
+    public class CookieResultContext : ResultContext<CookieAuthenticationHandler>
```

Comment:
It's a bit unfortunate to have to create another base class just to expose the options/scheme as first class properties :cry: